### DISCUSSION TITLE
Mount Sata SSD

### DISCUSSION THREAD

**Discussion by martinbone** [Joshua-Riek/ubuntu-rockchip#464] [Q&A]
Does anyone know how I can change the boot environment to enable mounting SATA SSD? eg. In ubuntuEnv.txt ... something like overlays=sata-ssd

I've already flashed the correct .img to my SPI to enable sata SSD.

**Comment by martinbone** [comment]
Found answer somewhere else. It's just overlays=sata    Ubuntu now running from my sata SSD   : )

**Reply by jigsurfer** [reply]
Hi!

Im currently stuck here. Coud you let me know which file and location ti put option: Overlays=...

I tried all solutions from overlays.txt & ubuntuEnv.txt

But lsblk never show up SSD from nvme slot

Best

**Reply by cybernextgen** [reply]
Same problem. Seems, that nor overlays=sata overlays=ssd-sata not working on OrangePi 5 at least in latest release.

**Comment by cybernextgen** [comment]
1. add to end of ```/boot/firmware/ubuntuEnv.txt``` next line: ```overlays=orangepi-5-sata```
2. update the bootloader with ```sudo u-boot-install-sata-mtd /dev/mtdblock0```
3. reboot and check output of ```lsblk```
